opus/application/static_media/css/opus.css

```diff
@@ -886,7 +909,7 @@ a:-webkit-any-link, a::any-link {
 .op-button-disabled {
     pointer-events: none;
     cursor: default;
-    opacity: 0.1;
+    opacity: 0.25 !important;
```

Comment:
How about 0.35? I tried it on a few different browsers and OSes and that seems to work better for me. It's still a big difference from the ~0.6 when it isn't disabled.